### PR TITLE
Feature merge data from external file

### PR DESCRIPTION
**Pull request by LokQiJun**
Implemented feature to merge a save file given file path and add it to list of current data

### CODE REVIEW

File: src/main/java/seedu/moneygowhere/storage/LocalStorage.java [hunk 1]
```diff
@@ -145,14 +205,89 @@ private Expense createExpense(Node node) throws LocalStorageLoadDataInputError {
         return new Expense(name, dateTime, description, amount, category, remark, currency);
     }
 
+    /**
+     * This method takes in a target node and convert it into a Target object.
+     *
+     * @param node containing information from a target
+     * @return a target created with data by node
+     */
```
Comment (xzynos) [inline src/main/java/seedu/moneygowhere/storage/LocalStorage.java:213]: I believe that your Javadoc comments may not be following the form defined in the coding standard. Do take a look at the coding standard at https://se-education.org/guides/conventions/java/basic.html

**Review by xzynos — COMMENTED** [review]
The code looks alright but do take a look at the coding standard with regards to the Javadoc comments

File: src/main/java/seedu/moneygowhere/storage/LocalStorage.java (hunk 1 above)
Comment (LokQiJun) [reply src/main/java/seedu/moneygowhere/storage/LocalStorage.java]: Noted. Will work on it.

File: docs/UserGuide.md [hunk 2]
```diff
@@ -127,6 +127,17 @@ Example of usage:
...
+### Merge file: `Merge-File`
+Merge save file from an external source given path to the save file
+
+Syntax: `Merge-File [-p PATH_STRING]`
+
+* `RECURRING_PAYMENT_INDEX` is an integer value. If this argument is provided, MoneyGoWhere will only display the specified recurring payment.
+
+Example of usage:
+
+* `merge-file -p "C:\Users\the_d\Downloads\expenses.txt"`
```
[the remark this answers is not in the document]
Comment (LokQiJun) [reply docs/UserGuide.md]: It should be a xml file. Thanks for pointing it out

**Review by xzynos — APPROVED** [review]
The code looks good!

**Pull request merged**